src/uct/ib/rc/base/rc_ep.h

```diff
@@ -202,16 +202,22 @@ struct uct_rc_ep {
     ucs_list_link_t     list;
     ucs_arbiter_group_t arb_group;
     uct_rc_fc_t         fc;
+    uint16_t            atomic_mr_offset;
     uint8_t             path_index;
 };
 
 UCS_CLASS_DECLARE(uct_rc_ep_t, uct_rc_iface_t*, uint32_t, const uct_ep_params_t*);
 
 
 typedef struct uct_rc_ep_address {
+    uint8_t          flags;
     uct_ib_uint24_t  qp_num;
 } UCS_S_PACKED uct_rc_ep_address_t;
 
+enum {
+    UCT_RC_ADDR_HAS_ATOMIC_MR = UCS_BIT(0),
```

Comment:
no need for `,` in the end

Reply:
comma after last enum member is consensual code style

Reply:
afair, it is allowed in C99, but not in C89. Typically we are trying to avoid that thru the code

Reply:
yes, we try to avoid comman for last elem, even though we actually require c99 for other things
@Artemy-Mellanox can you pls remove and squash?